model/tasks/PrintBooklet.php

```diff
@@ -52,6 +49,26 @@ public function __construct()
         $this->bookletClassService = BookletClassService::singleton();
     }
 
+    /**
+     * @param array $params
+     *
+     * @return common_report_Report
+     * @throws common_exception_MissingParameter
+     * @throws BookletExporterException
+     */
+    public function __invoke($params)
+    {
+        $report = parent::__invoke($params);
```

Comment:
We should try/catch any exception that is not covered deeper, so to delete the booklet in this case.

Reply:
Fixed